specs/eventing/data-plane.md

```diff
@@ -1,125 +1,183 @@
+which receive an error when probing with HTTP OPTIONS SHOULD proceed using the
+HTTP POST mechanism.
+
+### Event Acknowledgement and Delivery Retry
+
+Event recipients MUST use the HTTP response code to indicate acceptance of an
+event. The recipient SHOULD NOT return a response accepting the event until it
+has handled the event (processed the event or stored it in stable storage). The
+following response codes are explicitly defined; event recipients MAY also
+respond with other response codes. A response code not in this table SHOULD be
+treated as a retriable error.
+
+| Response code | Meaning                                           | Retry | Delivery completed | Error |
+| ------------- | ------------------------------------------------- | ----- | ------------------ | ----- |
+| `1xx`         | (Unspecified)                                     | No\*  | No\*               | Yes\* |
+| `200`         | [Accepted, event in reply](#derived-reply-events) | No    | Yes                | No    |
+| `202`         | Event accepted                                    | No    | Yes                | No    |
+| other `2xx`   | (Unspecified)                                     | No    | Yes                | No    |
+| `3xx`         | (Unspecified)                                     | No\*  | No\*               | Yes\* |
+| `400`         | Unparsable event                                  | No    | No                 | Yes   |
+| `404`         | Endpoint does not exist                           | Yes   | No                 | Yes   |
+| `409`         | Conflict / Processing in progress                 | Yes   | No                 | Yes   |
+| `429`         | Too Many Requests / Overloaded                    | Yes   | No                 | Yes   |
+| other `4xx`   | Error                                             | No    | No                 | Yes   |
+| `5xx`         | Error                                             | Yes   | No                 | Yes   |
+
+\* Unspecified `1xx`, `2xx`, and `3xx` response codes are **reserved for future
+extension**. Event recipients SHOULD NOT send these response codes in this spec
+version, but event senders MUST handle these response codes as errors or success
+as appropriate and implement described success or failure behavior.
+
+Recipients MUST be able to handle duplicate delivery of events (for example, via
+idempotency or tracking event delivery state) and MUST accept delivery of
+duplicate events, as the event acknowledgement could have been lost in return to
+the sender. As specified in the
+[CloudEvents specification](https://github.com/cloudevents/spec/blob/v1.0.1/primer.md#id),
+event recipients MUST use the
+[`source` and `id` attributes](https://github.com/cloudevents/spec/blob/v1.0.1/spec.md#required-attributes)
+to determine duplicate events if needed. This specification does not describe
+state requirements for recipients which need to detect duplicate events. (In
```

Comment:
Why the parens on the last sentence? It doesn't seem to be related to the dup discussion